components/introduction/modules/ROOT/pages/index.adoc

```diff
@@ -1,8 +1,82 @@
-= Overview
+= About SKALE Network
 
-Here are a few notes about getting started with SKALE chains.
+== What is SKALE Network 
 
-== SKALE Network Basics
+The SKALE Network is an open-source, Ethereum-native multichain network built to scale Ethereum dApps with a focus on high-throughput, fast finality, and zero gas fee transactions via the native gas token sFuel.
+
+== What problems does SKALE solve? 
+
+* `Poor web3 user experience`: 
+** end-users required to pay gas fees
+** low speed and latency
+** connectivity and messaging between chains
+* `Scalability Trilemma`
+** speed
+** security
+** decentralization
+* `Strict and ineffective blockchains`
+** Lack of customization and control
+** Lack of horizontal scaling
+** Block gas limits and gas costs limiting what and how data/transactions are put on-chain
+
+== What does SKALE offer?
+
+* `Zero gas fees` - by design, developers and end-users never pay for transactions.
+* `Instant finality` - the fastest chain to finality. Time to finality = blocktime.
+* `High performance`
+** `Throughput` - Baseline of 400 transactions per second (tps) and up to 700 tps at peak
+** `High block gas limit` - 280 million block gas limit allowing optimal throughput and complex transactions/smart contracts to be executed, like xref:tools::skale-specific/ml.adoc[on-chain Machine Learning].
+* `Purpose-built chains`- app-chains and hub-chains for customization and and horizontal scaling
+* `Native bridge` - Fast flexible, offering 18 second finality between chains. Direct support for popular token standards such as ERC-20, ERC-721, and ERC-1155 with additional flexibility for passing any arbitrary data through the low- level messaging protocol
+* `Native oracle` - decentralized and transaction free oracle enabling developers to read off-chain data
+* `Decentralization and fault tolerance` - chains supported by 16 randomly chosen and rotated SKALE Validator nodes, with a BFT tolerance of 1/3.
+* `Layered Blockchain Security` - unique hybrid Layer 1/Layer 2 architecture that partially derives security from Ethereum and with the performance and decentralization of a Layer 1.
+
+The Network enables dApp developers to access validator-operated nodes'compute power and storage to run fast, containerized, EVM-compatible chains called SKALE Chains.
+
+SKALE chains can exchange transactions or messages with other SKALE Chains and Ethereum-based networks through the native validator secured bridge (IMA), allowing dApp developers to use SKALE for powering their dApps while enabling token and message transfers to and from Ethereum or other chains.
```

Comment:
Suggested Change

...other SKALE Chains and the Ethereum Mainnet through the native validator secured bridge (IMA)